core/metadata.js

```diff
@@ -110,16 +112,27 @@ function assignIdUnix (doc /*: * */) {
   doc._id = doc.path
 }
 
-// Build an _id from the path for OSX (HFS+ file system):
+// Build an _id from the path for macOS, assuming file system is either APFS
+// or HFS+.
+//
+// APFS:
+// - case preservative, but not case sensitive
+// - unicode normalization preservative, but not sensitive
+//
+// HFS+:
 // - case preservative, but not case sensitive
 // - unicode NFD normalization (sort of)
```

Comment:
`// - unicode normalization insensitive, converts to NFD (sort of) automatically`

wdyt?